deploy/charts/cert-manager/templates/_helpers.tpl

```diff
@@ -34,10 +34,40 @@ Create the name of the service account to use
 {{- end -}}
 {{- end -}}
 
+{{/*
+Create the default PodDisruptionBudget to use
+*/}}
+{{- define "podDisruptionBudget.spec" -}}
+{{- if and .Values.podDisruptionBudget.minAvailable .Values.podDisruptionBudget.maxUnavailable }}
+{{- fail "Cannot set both .Values.podDisruptionBudget.minAvailable and .Values.podDisruptionBudget.maxUnavailable" -}}
```

Comment:
Why do we validate this here?
We don't validate any of the other resources, can't we let the apiserver figure out that something is wrong here?
I would opt for removing these helper functions and just inlining them instead.

Reply:
Wouldn't it fail now if I define something like this?!:
```
podDisruptionBudget
  enabled: false
  minAvailable: 1
  maxUnavailable: 1
```
Even if it is disabled by `enabled: false` since the helper doesn't care for this flag.

Reply:
Even if you don't enable it, the config is still wrong. No reason for it to be there. I don't think that's a valid reason for not validating.

Reply:
I'm rooting for @inteon suggestions.

@rouke-broersma i think you missed my point, anyway this wasn't a reason for not validating. Also @inteon wasn't suggesting to remove the validation completely, but move it into the templates itself.

Reply:
I changed it so we have a default of `minAvailable: 1` when enabled and nothing else is set. When either max or min is set, those values are used without defaults.